src/libcollections/string.rs

```diff
@@ -964,7 +961,7 @@ pub fn as_string<'a>(x: &'a str) -> DerefString<'a> {
     DerefString { x: as_vec(x.as_bytes()) }
 }
 
-#[unstable(feature = "collections", reason = "associated error type may change")]
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
Oh wow I totally missed that! I think that it may actually be appropriate in this case as `String::from_str` can never fail. It would in theory be more appropriately represented as `Result<String, Void>`, but adding `Void` probably isn't super well motivated by this use case.